README.md

```diff
@@ -26,7 +26,7 @@ The project is currently alpha. While no breaking API changes are currently plan
 * [Releases](#releases)
 * [Troubleshooting](#troubleshooting)
     * [Helm Error: no available release name found](#helm-error-no-available-release-name-found)
-
+    * [NFS volume mount failure: wrong fs type](#nfs-volume-mount-failure-wrong-fs-type)
```

Comment:
This change is unrelated to this PR.

Reply:
That's right. I'll move this to a separate branch.

Reply:
Done.